torch/_inductor/fx_passes/mkldnn_fusion.py

```diff
@@ -1048,7 +1048,10 @@ def forward(self, x):
 
         the above's packed weight nodes are duplicate if two linear calls have same input size.
         """
-        if not (torch.backends.mkldnn.enabled and torch.backends.mkldnn.is_available()):
+        if not (
+            getattr(torch.backends.mkldnn, "enabled", False)
```

Comment:
```
  Error (MYPYINDUCTOR) [attr-defined]
    Module has no attribute "enabled"

        1048  |
        1049  |        the above's packed weight nodes are duplicate if two linear calls have same input size.
        1050  |        """
    >>> 1051  |        if not (torch.backends.mkldnn.enabled and torch.backends.mkldnn.is_available()):
    ```

Reply:
This is due to this bit of code:

https://github.com/pytorch/pytorch/blob/44b98c09ca90ce4529d1a9792b01a0e8afe5345e/torch/backends/mkldnn/__init__.py#L86-L91

In this case, I would resolve it similarly to the dynamo config attributes:

https://github.com/pytorch/pytorch/blob/44b98c09ca90ce4529d1a9792b01a0e8afe5345e/torch/backends/mkldnn/__init__.py#L86-L91

i.e. we can declare `enabled` as a global of `torch.backends.mkldnn` during TYPE_CHECKING. (No need for a separate `.py.in` file here since we only have one attribute)